Lagrange.OneBot/Core/Network/Service/HttpPostService.cs

```diff
@@ -99,12 +96,25 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
     private async Task HeartbeatLoop(CancellationToken token)
     {
         var interval = TimeSpan.FromMilliseconds(_options.HeartBeatInterval);
+        Stopwatch sw = new();
+
+        if (!_options.HeartBeatEnable || _options.HeartBeatInterval <= 0)
+        {
+            await Task.Delay(Timeout.Infinite, token);
+        }
+
```

Comment:
In ForwardWebSocketService I've elevated the judgment to the outside, so that the Task doesn't start when no heartbeat is needed, which I think is better.

Reply:
dcae33347a7357d59debb07e9a1bd5e5c52733e0